[DOCS-99] Include Attestation Letter (#113)
cobalthq/cobalt-product-public-docs: content/en/Getting started/What to Expect/report-contents.md

Our pentest reports include what you need to further secure your systems.
{{% /pageinfo %}}

We provide following types of pentest reports:

{{% pentest-report-types %}}

> Our Full Report + Finding Details includes all of the following sections. With one exception,
> our other reports leave out some of these sections. If you've purchased an appropriate
> {{% ptaas-tier %}}, you can customize what you see in any of these reports.
> 
> We provide an Attestation Letter as a one-page report that you can share with external
> stakeholders such as prospects or customers. For more information, you're welcome to download
> this [sample Attestation Letter](/gsg/Sample_Attestation_Letter.pdf) for a web app.

Author: As noted by @yogipetkar, 

1. "We do not allow customization of Attestation Letter. Second sentence is not clear, if possible please simplify it."
2. The sample attestation letter is still in work; I may need to update the PDF file accordingly.

My response:

- Quite right, the second sentence is not clear. In this comment, I'm proposing that we delete it.
- I hope my other updates improve the clarity of this page. (I see your concern @yogipetkar , so I may rewrite this section again)
- I won't "merge" this change to production until the Attestation Letter is final. (It should be easy enough for me to update the PDF.)
- I'll "push" this change, so we can both see the impact on the build.

```suggestion
Our Full Report + Finding Details includes all of the following sections. If you've purchased an appropriate
{{% ptaas-tier %}}, you can customize what you see in all but the Attestation Letter.

The Attestation Letter is a one-page report that you can share with external stakeholders 
such as prospects or customers. For more information, download this [sample Attestation
Letter](/gsg/Sample_Attestation_Letter.pdf) for a web app.
```
